README.md

```diff
@@ -286,11 +286,12 @@ Install **[ale][vim-1]**. And add these lines to your `.vimrc` file.
 let g:ale_linters = {
 \   'javascript': ['standard'],
 \}
+let g:ale_fixers = {'javascript': ['standard']}
 ```
 
-This sets standard as your only linter for javascript files and so prevents conflicts with eslint.
+This sets standard as your only linter and fixer(on save) for javascript files and so prevents conflicts with eslint.
 
-For automatic formatting on save, add these lines to `.vimrc`:
+For automatic formatting on save, add these lines to `.vimrc` if you are not using **[ale][vim-1]**:
```

Comment:
I'd remove these altogether and just recommend using `ale`. The old method described here is pretty unusable anyway, since it locks up Vim for a couple of seconds every save!

Reply:
Instead consider changing that to something like

> For automatic formatting on save, use ale's `g:ale_lint_on_save` setting:
>
> ```vim
> let g:ale_lint_on_save=1
> ```